### ISSUE TITLE
Unable to load ADAPT map with sunpy>=6.0.0

### ISSUE THREAD

**Issue by NyankoSong** [sunpy/sunkit-magex#66]
### Describe the bug

Hi!
I'm here just to report this bug. I have downgraded `sunpy` to 5.1.4 and everything works fine now.
```
MultipleMatchError: Too many candidate types identified ([<class 'sunpy.map.sources.adapt.ADAPTMap'>, <class 'sunkit_magex.pfss.map.ADAPTMap'>]). Specify enough keywords to guarantee unique type identification.
```

### To Reproduce

```python
import sunpy.map

# this works fine
sunpy.map.Map('adapt40311_03k012_202201010000_i00005600n1.fts.gz', hdus=0) # random adapt map
```

```python
from sunkit_magex.pfss.utils import load_adapt

# MultipleMatchError: Too many candidate types identified ([<class 'sunpy.map.sources.adapt.ADAPTMap'>, <class 'sunkit_magex.pfss.map.ADAPTMap'>]). Specify enough keywords to guarantee unique type identification.
load_adapt('adapt40311_03k012_202201010000_i00005600n1.fts.gz')
```
```python
from sunkit_magex.pfss.utils import load_adapt
import sunpy.map

# MultipleMatchError: Too many candidate types identified ([<class 'sunpy.map.sources.adapt.ADAPTMap'>, <class 'sunkit_magex.pfss.map.ADAPTMap'>]). Specify enough keywords to guarantee unique type identification.
sunpy.map.Map('adapt40311_03k012_202201010000_i00005600n1.fts.gz', hdus=0)
```

### System Details

sunpy=6.0.1
astropy=6.1.2
sunkit-magex=1.0.0
python=3.11.8
OS: Ubuntu 24.04 LTS

### Installation method

conda

**Comment by nabobalis** [comment]
Thanks for the report @NyankoSong.

I think the problem is that we forgot to add an escape for the new Map class in sunpy 6.0.0.

The fix is hopefully in the open PR that removes that class from this library. 

For now, we will have to patch sunpy to prevent this from erroring and getting out a bugfix for 6.0.0

**Comment by nabobalis** [comment]
This has been patched in sunpy 6.0.1 and https://github.com/sunpy/sunkit-magex/pull/63 fixes it in sunkit-magex.
